server/v2/cometbft/abci.go

```diff
@@ -30,6 +31,9 @@ import (
 var _ abci.Application = (*Consensus[transaction.Tx])(nil)
 
 type Consensus[T transaction.Tx] struct {
+	// legacy support for gRPC
+	grpcQueryDecoders map[string]func(requestBytes []byte) (gogoproto.Message, error)
```

Comment:
We need to populate this from here: https://github.com/cosmos/cosmos-sdk/pull/20808/files#r1658607864